instaloader/__main__.py

```diff
@@ -268,6 +268,8 @@ def main():
                         help="Do not download regular posts.")
     g_prof.add_argument('--no-profile-pic', action='store_true',
                         help='Do not download profile picture.')
+    g_prof.add_argument('--slide', action='store',
```

Comment:
```suggestion
    g_post.add_argument('--slide', action='store',
```

This belongs to group *What to download of each Post* rather than *What to Download of each Profile*.